CODE_REVIEW.md

```diff
@@ -0,0 +1,46 @@
+
+# Code Review
+
+## Problems 
+1. Potential memory leak from observable in `ngOnInit()` in `libs/books/feature/src/lib/book-search/book-search.component.ts`. The observable should be unsubscribed from using `ngOnDestroy()` **--Fixed**
+
+2. Files with different functions are grouped together in `libs/books/data-access/src/lib/+state`. Actions, reducer and selector files should be grouped according to their function and their reference undated everywhere
+
+3. No type tags added in `nx.json` and defined in `tslint.json`. Tags should be added to control folder access between apps in a mono repo
```

Comment:
Can you elaborate on what you mean by "folder access"? What sort of tags would you add?

Reply:
The words is was looking for is "dependence tags", which allow you to specify the libraries an application can depend on. For example, the api (Nest) application will only depend on (or be able to import from) the libs folder that contain "api" tag.